.circleci/config.yml

```diff
@@ -12,7 +12,6 @@ jobs:
             source /usr/local/share/virtualenvs/tap-mailchimp/bin/activate
```

Comment:
Code change added mistakenly as part of Commit Link (https://github.com/singer-io/tap-mailchimp/commit/da7b0f15d9d5b51081d4d75345c7952645a277af) are reverted in this PR